Adds `graphics-document`, `graphics-object`, and `graphics-symbol` `aam-graphics` roles.

younisdev opened the pull request:
<!-- 🙌 Thanks for contributing to MDN Web Docs. 🙌 -->

<!--
Add details below to help us review your pull request (PR).
Explain your changes and link to a related issue or pull request.
Your PR may be delayed or closed if you don't provide enough information.
-->

### Description

<!-- ✍️ Summarize your changes in one or two sentences. -->
Add`graphics-document`, `graphics-object`, and `graphics-symbol` `aam-graphics` roles.
### Motivation

<!-- ❓ Why are you making these changes and how do they help readers? -->
These changes ensure that the reader are aware of the 3 graphical am-graphics roles. And ensures that MDN is up to date!
### Additional details

<!-- 🔗 Link to release notes, browser docs, bug trackers, source control, or other resources. -->
None.
### Related issues and pull requests

<!-- 🔨 If this fully resolves a GitHub issue, use "Fixes #123" -->
<!-- 👉 Highlight related pull requests using "Relates to #123" -->
<!-- ❗ If another pull request must be merged first, use "**Depends on:** #123" -->
Fixes #28707 
<!-- 🔎 After submitting, the 'Checks' tab of your PR shows the build status. -->

younisdev committed:
Adds graphics-document, 'graphics-object', and graphics-symbol ARIA roles.

younisdev committed:
Merge branch 'mdn:main' into update-5

Josh-Cena commented:
Reminder that you need to update the parent page, Web/Accessibility/ARIA/Reference/Roles

younisdev committed:
Fix link typi

younisdev commented:
> Reminder that you need to update the parent page, Web/Accessibility/ARIA/Reference/Roles

Should I add these as a separate "Graphics roles" section?

Josh-Cena commented:
Not sure

younisdev committed:
Add graphics role aria-type

scottaohara commented on files/en-us/web/accessibility/aria/reference/roles/graphics-document_role/index.md:
```diff
@@ -0,0 +1,56 @@
+- `graphics-object`
+  - : The `graphics-document` role is self-contained and retains its meaning when separated from the surrounding content.
+
+## Examples
```
this probably isn't the best example, since these roles shouldn't even be needed for these svg elements.

scottaohara reviewed (changes requested):
just leaving a quick review.

i don't have time to dig into these further, but i'm mostly concerned that the narrative here is telling people to use these roles on elements that don't need them, and including fallback roles unnecessarily.

younisdev committed:
Polish Aria graphics roles.

younisdev committed:
Reverse a mistake.

Reorganize sections on roles in ARIA documentation, replacing 'Graphics roles' with 'Abstract roles' and updating descriptions accordingly.

younisdev commented:
> just leaving a quick review.
> 
> i don't have time to dig into these further, but i'm mostly concerned that the narrative here is telling people to use these roles on elements that don't need them, and including fallback roles unnecessarily.

Thanks for the review! I've added a note to all the aam-graphics that clearly states these docs aren't a part of the core WAI-ARIA spec. I also removed the fallback note , and  reworded sections to be less prescriptive.

younisdev committed:
Update index.md

younisdev committed:
Improve examples for aam graphics roles

younisdev committed:
Improve examples for aam graphics roles

younisdev committed:
Merge branch 'update-5' of https://github.com/younisdev/content into update-5

younisdev committed:
Modify graphics-document aam role example to show non linear flow with some tone adjustments

younisdev committed:
Modify graphics-object aam role example to show an accurate thematic map of all continents

younisdev committed:
Modify graphics-symbol aam role example to show atomic parking space icons

younisdev committed:
Fix CI failure

younisdev committed:
try fix the cl issue

younisdev committed:
Delete package-lock.json

younisdev committed:
Merge branch 'mdn:main' into update-5
